cloud/pkg/nodeupgradejobcontroller/manager/common.go

```diff
@@ -17,46 +17,61 @@ limitations under the License.
 package manager
 
 import (
-	"k8s.io/apimachinery/pkg/runtime"
-	"k8s.io/apimachinery/pkg/watch"
-	"k8s.io/klog/v2"
+	"k8s.io/client-go/tools/cache"
 )
 
-// Manager define the interface of a Manager, NodeUpgradeJob Manager implement it
-type Manager interface {
-	Events() chan watch.Event
```

Comment:
Do we need use channel to cache events, or just use AddFunc/UpdateFunc/DeleteFunc to process event directly